### DISCUSSION TITLE
Unable to install dependencies after cloning  react play

### DISCUSSION THREAD

**Discussion by Amanaakash** [reactplay/react-play#1113] [Q&A]
Hey , i  am unable to install the dependencies as instructed in the ReadMe file of react play , i ran (npm install) but it doesn't work throws error which says (npm ERR! enoent This is related to npm not being able to find a file.)

**Comment by atapas** [comment]
Can you please come on discord where we can resolve it interactively?

https://discord.gg/UfYj4MvW9A
